build/Dockerfile

```diff
@@ -1,12 +1,30 @@
-FROM golang:1.13.8 AS build
-
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+# NOTE: using registry.access.redhat.com/rhel8/go-toolset does not work (user is requested to use registry.redhat.io)
+# NOTE: using registry.redhat.io/rhel8/go-toolset requires login, which complicates automation
+# NOTE: since updateBaseImages.sh does not support other registries than RHCC, update to RHEL8
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/devtools/go-toolset-rhel7
+FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.13.4-18 AS build
```

Comment:
AFAIK this image contains vulnerabilities

Reply:
@benoitf the same image image is used in the che-operator - https://github.com/eclipse/che-operator/blob/master/Dockerfile#L29

Reply:
latest is `1.13.4-23.1592916276` so it should be updated for operator as well

Reply:
sure I can update it in the current repo, but who should own the updates of images at this point? Looks like there is no mechanism in place for identifying obsolete ubi images that are used by product

Reply:
downstream we have already updated http://pkgs.devel.redhat.com/cgit/containers/codeready-workspaces-operator/tree/Dockerfile?h=crw-2.2-rhel-8#n16 because I have bots that can commit changes directly.

upstream we can't have bots autopushing changes because ECA and CODEOWNERS. 

So instead I intended to generate PRs against upstream but that got stalled.